### PR TITLE
Make SPI speed configurable for ILI9341 displays

### PR DESCRIPTION
**Pull request by renzenicolai**

### CODE REVIEW

File: firmware/components/driver_display_ili9341/Kconfig [hunk 1]
```diff
@@ -19,6 +19,11 @@ menu "Driver: ILI9341 LCD display"
 	config PIN_NUM_ILI9341_BACKLIGHT
 		depends on DRIVER_ILI9341_ENABLE
 		int "GPIO pin used for backlight"
+		
+	config DRIVER_ILI9341_SPI_SPEED
+		depends on DRIVER_ILI9341_ENABLE
+		int "SPI clock speed"
+		default 1000000
```
Comment (tjclement) [inline firmware/components/driver_display_ili9341/Kconfig]: Why is the new default 1MHz when previously it was set to 40MHz? Seems a bit drastic to me

**Review by tjclement — CHANGES_REQUESTED** [review]

File: firmware/components/driver_display_ili9341/Kconfig (hunk 1 above)
Comment (renzenicolai) [reply firmware/components/driver_display_ili9341/Kconfig]: Some devices are less capable than others. While 40MHz should work I've seen some issues with crappy breadboard constructions. 1MHz is the failsafe default.

**Comment by renzenicolai** [comment]
@tjclement Please have a look at my comment and rethink your answer :-)

**Comment by tjclement** [comment]
Yep, I see your point. Let's add "range 100000 40000000" to the config item, so we know in the future what sort of values are reasonable here.

**Comment by renzenicolai** [comment]
@tjclement I've added the range and I've set the default value to 40MHz

**Review by tjclement — APPROVED** [review]
Very nice

**Pull request merged**